integration/test/app/kubeconfig/kubeconfig_test.go

```diff
@@ -189,7 +190,7 @@ func TestAppWithKubeconfig(t *testing.T) {
 				Name:      chartOperatorName,
 				Namespace: namespace,
 				Labels: map[string]string{
-					label.AppOperatorVersion: key.AppOperatorVersion(),
```

Comment:
operator doesn't have any usage for`AppOperatorVersion`, can we delete this function?

Reply:
Thanks, good catch. Done.